src/log-0.0.1.js

```diff
@@ -171,6 +172,13 @@ LogJS.api.Log = function Log(_opts) {
     self.getName = function() {
         return name;
     };
+    
+    self.setTimeStampCreator = function (fn) {
```

Comment:
It would be nice to have small documentation or reference about this TimeStampCreator